raiden-dapp/tests/unit/store.spec.ts

```diff
@@ -221,5 +221,13 @@ describe('store', () => {
         store.getters.channels('0xd0A1E359811322d97991E03f863a0C30C2cF029C')
       ).toEqual([]);
     });
+
+    test('when token has no channels token getter should return an empty array', () => {
+      const channels = {
+        '0xd0A1E359811322d97991E03f863a0C30C2cF029C': {}
+      };
+      store.commit('updateChannels', channels);
+      expect(store.getters.tokens).toEqual([]);
```

Comment:
Before the change, it was returning an empty accumulator instance that had an empty string as an address and only the initial values. This empty address was causing the issues with the $blockie.